lib/series/seriesmanager.py

```diff
@@ -234,7 +239,9 @@ async def read_from_disk(cls):
             series_data = data.get('series')
             if series_data is not None:
                 for s in series_data:
-                    cls._series[s.get('name')] = Series.from_dict(s)
+                    await cls._add_series(s)
+                    # cls._series[s.get('name')] = Series.from_dict(s)
+                    # cls._series[s.get('name')]
```

Comment:
Why are these lines commented out?

Reply:
Short reminders for myself. I will clean it up before releasing a new version